core/src/main/java/jenkins/util/groovy/AbstractGroovyViewModule.java

```diff
@@ -0,0 +1,48 @@
+package jenkins.util.groovy;
+
+import groovy.lang.GroovyObjectSupport;
+import lib.FormTagLib;
+import lib.LayoutTagLib;
+import org.kohsuke.stapler.jelly.groovy.JellyBuilder;
+import org.kohsuke.stapler.jelly.groovy.Namespace;
+import lib.JenkinsTagLib;
+
+/**
+ * Base class for utility classes for Groovy view scripts
+ * <p>
+ * Usage from script of a subclass, say ViewHelper:
+ * <p>
+ * <tt>new ViewHelper(delegate).method();</tt>
+ * <p>
+ * see <tt>ModularizeViewScript</tt> in ui-samples for an example how to use
+ * this class.
```

Comment:
Uh huh. Apparently never used in production code. I did test `ui-samples-plugin` against this snapshot and confirmed that `/ui-samples/ModularizeViewScript/` looks identical as against Jenkins 2.7.3, the baseline.